package.json

```diff
@@ -51,7 +50,7 @@
     "husky": "^7.0.4",
     "jest": "^27.4.7",
     "lint-staged": "^12.1.7",
-    "mongodb": "^3.6.7",
+    "mongodb": "~4.1.0",
```

Comment:
Will move this to `^4.0.0` once https://github.com/mongodb/node-mongodb-native/pull/3102 is published.